forge-gui/res/cardsfolder/upcoming/stingerback_terror.txt

```diff
@@ -0,0 +1,13 @@
+Name:Stingerback Terror
+ManaCost:2 R R
+Types:Creature Scorpion Dragon
+PT:7/7
+K:Flying
+K:Trample
+K:Plot:2 R
+S:Mode$ Continuous | Affected$ Card.Self | AddPower$ -X | AddToughness$ -X | Description$ CARDNAME gets -1/-1 for each card in your hand.
+SVar:X:Count$InYourHand
+SVar:BuffedBy:Card
+SVar:NeedsToPlayVar:Y LE6
+SVar:Y:Count$CardsInYourHand
```

Comment:
I don't see why you can't reuse the other SVar here

Reply:
I lifted this part of the script from [Grim Strider](https://scryfall.com/card/akh/94/grim-strider). [Geralf's Masterpiece](https://scryfall.com/card/soi/65/geralfs-masterpiece), the other card that has this debuff also uses this script structure. 
As for related debuffs, [Nyxathid](https://scryfall.com/card/jmp/259/nyxathid)'s doesn't reuse the `SVar` for applying the malus and the variable called upon by `NeedsToPlayVar` is rather involved. [Enemy of Enlightenment](https://scryfall.com/card/thb/92/enemy-of-enlightenment) does reuse its malus `SVar` though.
I wouldn't know where to begin to test which approach is right in which circumstance.

Also going by [Mogg Squad](https://scryfall.com/card/tmp/192/mogg-squad), shouldn't it be `SVar:AntiBuffedBy:Card` instead?

Reply:
please clean up those simple cases, it will be fine

Mogg Squad is different since it's about keeping stuff off the battlefield

Reply:
Fair enough: done. Duly noted otherwise.